docs/glueops-platform-administrator/configuration/export-logs.md

```diff
@@ -0,0 +1,292 @@
+---
+id: export-logs
+title: Exporting Logs from GlueOps
+---
+
+# Export/Ship your logs to an external sink
+
+:::danger
+These docs are beta.
+:::
+ 
+We use the [fluent-operator](https://github.com/fluent/fluent-operator) to ship your logs however the version we run is not always the latest so it's best to verify the correct CRDs to use.
+
+At a high level this doc will cover collecting logs, parsing/modify logs, and shipping logs.
+
+
+## Collecting Logs (INPUTS)
+
+```yaml title="Example to collect all container logs"
+apiVersion: fluentbit.fluent.io/v1alpha2
+kind: ClusterInput
+metadata:
+  labels:
+    logging.glueops.dev: 'glueops-core-primary' # needs to match the label defined in: ClusterFluentBitConfig
+  name: 'glueops-core-platform-cri-primary' # Must be unique among all objects of same type
+  namespace: glueops-core-fluent-operator
+spec:
+  tail:
+    db: /fluent-bit/tail/pos.db
+    dbSync: Full
+    memBufLimit: 100MB
+    parser: cri
+    path: /var/log/containers/*.log
+    readFromHead: false
+    refreshIntervalSeconds: 10
+    skipLongLines: true
+    #storageType: filesystem
```

Comment:
it might be useful to add a comment for why this is commented out

Reply:
@fernandoataoldotcom updated